src/ts/components/product/Product.ts

```diff
@@ -57,14 +37,34 @@ class Product {
     }
   }
 
+
+  handleAddProduct = (event: Event) => {
+    event.preventDefault();
+    const [productName, productPrice, productQuantity] = 
+      this.productInfoInputs.map((input: HTMLInputElement ) => input.value);
+    const product = {
+      productName: productName,
+      productPrice: +productPrice,
+      productQuantity: +productQuantity,
+    };
+
+    this.productInfo.validateProductInfo({ ...product });
+    showSnackbar(registerProductText({ ...product }));
+    this.productInfo.addProductList({ ...product });
+    this.productView.changeProductInfoInputEmpty();
+    this.productView.focusProductNameInput();
+    this.productView.addProduct({ ...product });
```

Comment:
```suggestion
    this.productInfo.validateProductInfo(product);
    showSnackbar(registerProductText(product));
    this.productInfo.addProductList(product);
    this.productView.changeProductInfoInputEmpty();
    this.productView.focusProductNameInput();
    this.productView.addProduct(product);
```
인자가 같고 함수 내부에서 사이드이펙트가 없다면 전개연산자로 넘겨줄 필요는 없을거같아요 👀